src/content/docs/release-notes/mobile-release-notes/xamarin-release-notes/xamarin-agent-004.mdx

```diff
@@ -0,0 +1,13 @@
+---
+subject: Xamarin agent
+releaseDate: '2024-03-06'
+version: 0.0.4
+downloadLink: 'https://www.nuget.org/packages/NewRelic.Xamarin.Plugin'
+---
+
+## New in this release
+
+* Added Offline Monitoring Feature: This new feature enables the preservation of harvest data that would otherwise be lost when the application lacks an internet connection. The stored harvests will be sent once the internet connection is re-established and the next harvest upload is successful.
+* Introduced setMaxOfflineStorageSize API: This new API allows the user to determine the maximum volume of data that can be stored locally. This aids in better management and control of local data storage.
+* Updated native iOS Agent: We've upgraded the native iOS agent to version 7.4.9, which includes performance improvements and bug fixes.
+* Updated native Android Agent: We've also upgraded the native Android agent to version 7.3.0 bringing benefits like improved stability and enhanced features.
```

Comment:
```suggestion
* Added offline monitoring feature: This new feature enables the preservation of harvest data that would otherwise be lost when the application lacks an internet connection. The stored harvests will be sent once the internet connection is re-established and the next harvest upload is successful.
* Introduced `setMaxOfflineStorageSize` API: This new API allows the user to determine the maximum volume of data that can be stored locally. This aids in better management and control of local data storage.
* Updated native iOS agent: We've upgraded the native iOS agent to version 7.4.9, which includes performance improvements and bug fixes.
* Updated native Android agent: We've also upgraded the native Android agent to version 7.3.0 bringing benefits like improved stability and enhanced features.
```